base/plumbing/DataStore.js

```diff
@@ -632,21 +632,18 @@ class Store {
 		}
 		// No PV ...but already in the store?
 		// Note: since we retain PVs (to maintain identity i.e. same item always has same PV) 
-		// this check is for a corner case (which does happen - e.g. saveAs()), 
-		// where the datastore has an object that wasn't entered via fetch(). 
+		// this check is for a corner case (which does happen - e.g. saveAs()),
+		// where the datastore has an object that wasn't entered via fetch().
 		// For items managed purely by fetch(), we have:
 		// - if the item is absent & no fetch in progress, fetch2() will start a fetch & return PV
 		// - if the item is absent but fetch in progress, fetch2() will return in-progress PV
 		// - if the item is present & fresh (within cache period) fetch2() will return the resolved PV
 		// - if the item is present & stale, fetch2() will start a refresh fetch, but return the old PV
 		const item = this.getValue(path);
 		// Note: falsy or an empty list/object is counted as valid, only null/undefined will trigger a fresh load.
-		const notHere = (item === null || item === undefined);
-		if (notHere) return null;
-		// make and store a PV
-		pv = new PromiseValue(item);
-		this.setFetchPV(path, pv);
-		return pv;
+		if (item == null) return null;
```

Comment:
Hello @roscoemcinerney Could you add some doc notes on the meaning & handling of `undefined` vs `null`? Thanks.